cmake-tool/helpers/rootserver.cmake

```diff
@@ -101,37 +97,33 @@ function(DeclareRootserver rootservername)
             else()
                 set(march rv64imafdc)
             endif()
-            if(UseRiscVBBL)
-                # Package up our final elf image into the Berkeley boot loader.
+            if(UseRiscVOpenSBI)
+                # Package up our final elf image into OpenSBI.
                 # The host string is extracted from the cross compiler setting
```

Comment:
I think we can drop the comment "_The host string is extracted from the cross compiler setting minus the trailing '-'_" from the code also, because this was something for Proxy-PK and all the CMake code block seem to be gone bow. There is not longer a "host" variable anywhere